crates/egui/src/containers/window.rs

```diff
@@ -356,6 +356,15 @@ impl<'open> Window<'open> {
         self
     }
 
+    /// If there is [`ScrollArea`], For each axis, should the containing area shrink if the content is small?
+    ///
+    /// Show the [`ScrollArea::auto_shrink`]
+    #[inline]
+    pub fn auto_shrink(mut self, auto_shrink: impl Into<Vec2b>) -> Self {
+        self.scroll = self.scroll.auto_shrink(auto_shrink);
```

Comment:
From the docs of `ScrollArea::auto_shrink`:

> * If `true`, egui will add blank space outside the scroll area.
> * If `false`, egui will add blank space inside the scroll area.

For a window we never want blank space outside the scroll area, so we should just change the default of `Window::scroll` to always have `.auto_shrink(false)`

Reply:
I didn't fully understand what you meant.

This doesn't seem to be clearly explained in `ScrollArea::auto_shrink`.


If we want to change the default value of `egui::Window`,

1. we can either change the default value of `ScrollArea::auto_shrink` to `false`,

OR

2. add an `auto_shrink` variable to `egui::Window`.

I am unsure of which option is the best approach.

Reply:
In `Window::new` do `scroll: ScrollArea::neither().auto_shrink(false)`